### PR TITLE
*: fix makefile && improve `show tables query` 

### PR DESCRIPTION
**Pull request by sundy-li**
Some small fixes in `show tables`

### CODE REVIEW

File: src/storages/system/systemstorage_tables.go [hunk 1]
```diff
@@ -56,8 +56,12 @@ func (storage *SystemTablesStorage) GetInputStream(session *sessions.Session, sc
 
 	// Block.
 	block := datablocks.NewDataBlock(cols)
-	if err := ctx.tablesFillFunc(dbName, block); err != nil {
-		return nil, err
+
+	dbNames := ctx.allDatabaseNames()
```
Comment (BohuTANG) [inline src/storages/system/systemstorage_tables.go]: Hello, 
SystemTables only return the tables which belongs to the current database(session.GetDatabases()).
allDatabaseNames used for?

Comment (sundy-li) [reply src/storages/system/systemstorage_tables.go]: I think `system.tables` shall return all tables which behaviors the same as ClickHouse.

Comment (sundy-li) [reply src/storages/system/systemstorage_tables.go]:
```
SELECT
    database,
    name
FROM system.tables
GROUP BY
    database,
    name

┌─database────┬─name───────────────────────────┐
│ system      │ asynchronous_metrics           │
│ system      │ merges                         │
│ system      │ clusters                       │
│ system      │ table_engines                  │
│ system      │ parts                          │
│ test_slhi60 │ as_foo                         │
│ test_slhi60 │ tab_str                        │
```

Comment (BohuTANG) [reply src/storages/system/systemstorage_tables.go]: I see, Thanks.
But this needs more things to be changed:
1. fillTablesFunc should return all the database tables in databases/databases_callback.go, no need allDatabaseNames() here.
BTW, *Context.go only for parameter passing by design :D
2. https://github.com/vectorengine/vectorsql/blob/master/src/planners/planner_show_tables.go#L27 
query := "select * from system.tables order by name asc"
change to 
query := "select * from system.tables"

3. Rewritten the ShowTables Planner in:
https://github.com/vectorengine/vectorsql/blob/master/src/executors/executor_show_tables.go#L12
3.1 Add FilterPlanner and OrderByPlanner, the final query is: "select * from system.tables where database=[currentdatabase] order name asc"

**Comment by sundy-li** [comment]
Maybe we should just rewrite the `AST`  rather than fill the planner manually.
```
query := fmt.Sprintf("select * from system.tables where `database` = '%s' order by name asc", ctx.session.GetDatabase())
ast, _ := parsers.Parse(query)

planner.SubPlan = planners.NewSelectPlan(ast)
planner.SubPlan.Build()
```

**Comment by BohuTANG** [comment]
@sundy-li 
> Maybe we should just rewrite the `AST` rather than fill the planner manually.
> 
> ```
> query := fmt.Sprintf("select * from system.tables where `database` = '%s' order by name asc", ctx.session.GetDatabase())
> ast, _ := parsers.Parse(query)
> 
> planner.SubPlan = planners.NewSelectPlan(ast)
> planner.SubPlan.Build()
> ```
Yeah.
The good is simple, and the bad is that planners package needs import sessions(This is not suggested, planners works with sessions is in optimizers in the feature).

By the way, it's better to rebase 3-commits to one golang-style, such as:
```
*: fix makefile && fix show tables ...
```
`*: `stands for this patch refer to many(more than 2)packages, if one will be:
```
package-name: commit-message,blabla...
```

Thanks.

**Pull request merged**